openstudiocore/src/model/PlanarSurface.hpp

```diff
@@ -218,6 +219,8 @@ class MODEL_API PlanarSurface : public ParentObject {
   /// Returns any solar hot water collectors associated with this surface.
   std::vector<ModelObject> solarCollectors() const;
 
+  /// Returns any generator photovoltaics associated with this surface.
+  std::vector<GeneratorPhotovoltaic> generatorPhotovoltaics() const;

```

Comment:
This is what is suggested by the data model and doing anything else would take a lot of model logic to enforce and introduce potential problems, thoughts?